target/linux/mediatek/image/filogic.mk

```diff
@@ -204,6 +204,25 @@ define Device/cetron_ct3003
 endef
 TARGET_DEVICES += cetron_ct3003
 
+define Device/confiabits_mt7981
+  DEVICE_VENDOR := Confiabits
+  DEVICE_MODEL := MT7981
+  DEVICE_DTS := mt7981b-confiabits-mt7981
+  DEVICE_DTS_DIR := ../dts
+  IMAGES := sysupgrade.bin
+  SUPPORTED_DEVICES += confiabits,mt7981
+  UBINIZE_OPTS := -E 5
+  BLOCKSIZE := 128k
+  PAGESIZE := 2048
+  IMAGE_SIZE := 65536k
+  KERNEL_IN_UBI := 1
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-ubi | check-size $$$$(IMAGE_SIZE)
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb3 kmod-mt7981-firmware
```

Comment:
`kmod-usb2`?  `kmod-usb3` is not enough?

Reply:
Good point, I check and if it works I remove the kmod-usb2 package.